screens/Settings/About/index.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import About from './index'
+
+describe('About', () => {
+  test('renders', async () => {
+    const { getByText } = render(<About />)
+
+    getByText('Application Version')
+    getByText('Build Version')
```

Comment:
Same issue with cypress where we cannot asset on the app version or build version at this point as it is undefined